requirements.txt

```diff
@@ -2,4 +2,8 @@
 
 -e git+git://github.com/egabancho/invenio-oauthclient.git@0ae2a4a5c89374dd9c1b7a2c22155b77e4b02325#egg=invenio-oauthclient
 
+-e git+git://github.com/inveniosoftware/invenio-search-ui#egg=invenio-search-ui
+
+-e git+git://github.com/inveniosoftware/invenio-collections#egg=invenio-collections
```

Comment:
Current `pypi` version of InvenioCollections it is not working with the latest version of InvenioDB because of the nested transactions.